src/main/java/psi/model/ListItem.java

```diff
@@ -0,0 +1,29 @@
+package psi.model;
+
+public class ListItem implements Item{
+	private Integer order;
```

Comment:
Not sure I remember what the order means... 
If it's an "ordered" flag, should it be a bool instead?
If it holds some ranks, should it be a list of integers...? Or maybe use some other structure than the list...

Reply:
It's not a flag, it's the item's number of order within the parent list. I could delete it I guess, I chose to let it  be for now because it seemed simpler this way.

Reply:
Hmm... feels a little un-natural for an item to know it's index within the container list, it's again dependence from the contained towards the container... 
Would rather delete it, let the list know the indexes 😁 
Just a feeling, again, ignore if your feeling is otherwise 😎

Reply:
It's a good point, thank you!